website/views/pages/imagine/higher-education.ejs

```diff
@@ -0,0 +1,100 @@
+<div id="higher-education" v-cloak>
+
+  <div purpose="hero-background" class="container-lg d-flex justify-content-center">
+
+    <div purpose="hero-container" class="d-flex flex-column align-items-center">
+      <div purpose="hero-text">
+        <h4 class="mb-2">Visibility and compliance in a single platform</h4>
+        <h1>Higher education meets simplified security</h1>
+        <p>Automate security workflows in a single application by creating or installing policies to identify which devices comply with your security guidelines.</p>
+        <div purpose="button-row" class="d-flex flex-sm-row flex-column justify-content-center align-items-center">
+          <a purpose="cta-button" href="/try-fleet/register?tryitnow">Try Fleet today</a>
+          <a @click="clickOpenChatWidget()" purpose="animated-arrow-button-red">Talk to an expert</a>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+  <div purpose="page-container" class="container-lg">
+    <div purpose="feature" class="d-flex flex-md-row flex-column-reverse justify-content-between mx-auto align-items-center">
+      <div class="d-flex flex-column">
+        <h3>Out-of-the-box CIS benchmark policies</h3>
+        <p>Colleges are often subject to strict regulatory compliance requirements. With Center for Internet Security (CIS) compliance policies in Fleet, cybersecurity best practices are at your fingertips.</p>
+        <div purpose="checklist" class="flex-column d-flex">
+          <p>Out of the box policies for Mac, Windows, and Linux.</p>
```

Comment:
Can you change this line to be consistent with the other places "out-of-the-box" is used?